content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -0,0 +1,119 @@
+
+Even within these two topologies, there is more than one way to configure a multicluster mesh.
+Which approach to use and how to configure it depends on the requirements of the application
+and on the features and limitations of the underlying cloud deployment platform.
+
+### Single control plane topology
+
+This multicluster configuration uses a single Istio control plane running on one of the clusters.
+The control plane's Pilot manages services on the local and remote clusters and configures the
+Envoy sidecars for all of the clusters. This approach works best in environments where all of
+the participating clusters have VPN connectivity so every pod in the mesh is reachable from anywhere
+else using the same IP address.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-vpn.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters with direct network access to remote pods over VPN"
+    >}}
+
+In this configuration, the Istio control plane is deployed on one of the clusters while all other
+clusters run a simpler remote Istio configuration which connects them to the single Istio control plane
+that manages all of the Envoy's as a single mesh. The IP addresses on the various clusters must not
+overlap and note that DNS resolution for services on remote clusters is not automatic.
+Users need to replicate the services on every participating cluster.
+You can find detailed steps to set up this kind of multicluster topology
+in our [single control plane with VPN instructions](/docs/setup/kubernetes/multicluster-install/vpn/).
+
+If setting up an environment with universal pod-to-pod connectivity is difficult or impossible,
+it may still be possible to configure a single control plane topology using Istio gateways and
+configuring Istio Pilot to enable location-aware service routing (a.k.a. split-horizon EDS).
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-split-horizon-eds.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters using single Istio control plane and Gateway to reach remote pods"
+    >}}
+
+In this configuration, the control plane forwards a request from a sidecar in one cluster to a service in
+the same cluster using the local service IP as usual. If the destination workload is running in a different cluster,
+the control plane uses the remote cluster Gateway IP to connect to the service instead.
+Visit our [single control plane with gateways example](/) (TODO ref: /docs/examples/multicluster/split-horizon-eds/)
+to experiment with this feature.
+
+Note that this approach still requires connectivity between the Istio control plane and the Kubernetes
+API servers on every cluster. If this is also difficult or impossible, a multiple control plane topology is
+probably a better alternative.
+
+### Multiple control plane topology
+
+In a multiple control plane configuration, each cluster has an identical Istio control plane
```

Comment:
I still think that this should be the first option as its nice to start with the easiest option (with some limitations) and then work your way towards the harder ones in the doc.